recipes/rescript-mode

```diff
@@ -0,0 +1,4 @@
+(rescript-mode
+ :fetcher github
+ :repo "jjlee/rescript-mode"
+ :branch "master")
```

Comment:
```suggestion
 :repo "jjlee/rescript-mode")
```
Not necessary, as long as this is your default branch (we prefer MELPA to build from the default branch)

Reply:
I read that in CONTRIBUTORS.org and didn't understand "the default branch"... but I guess this is referring to github having a concept of default branch?